### PR TITLE
Add PropertiesContainer ModelView component

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Currently this just supports string values since that's what the core component supports. Ideally I'd like to support full components for the value part so we can get special things such as hyperlinks and buttons - will have to think about that a bit though so getting this out now to reduce overall complexity.

### CODE REVIEW

File: src/sql/azdata.proposed.d.ts [hunk 1]
```diff
@@ -305,6 +306,41 @@ declare module 'azdata' {
...
+	 */
+	export interface PropertiesContainerComponentProperties {
+		/**
+		 * The properties to display
+		 */
+		displayProperties?: DisplayProperty[];
+		/**
+		 * Whether the component is currently loading
+		 */
+		loading?: boolean;
```
Comment (Charles-Gagnon) [inline src/sql/azdata.proposed.d.ts:341]: I might also come back and separate out the loading part from the core component too. It's kind of weird to have this be the only one that has it built-in and I'm thinking it'd be better to just have the dashboard use the loading component separately

File: src/sql/workbench/browser/modelComponents/propertiesContainer.component.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	Component, Input, Inject, ChangeDetectorRef, forwardRef,
+	ViewChild, ElementRef, OnDestroy
+} from '@angular/core';
+
+import * as azdata from 'azdata';
+import { ComponentBase } from 'sql/workbench/browser/modelComponents/componentBase';
+import { IComponent, IComponentDescriptor, IModelStore } from 'sql/platform/dashboard/browser/interfaces';
+import { PropertiesContainer, DisplayProperty } from 'sql/base/browser/ui/propertiesContainer/propertiesContainer.component';
+
+export interface DisplayPropertyConfig {
```
Comment (kisantia) [inline src/sql/workbench/browser/modelComponents/propertiesContainer.component.ts:16]: what is this used for?

Comment (Charles-Gagnon) [reply src/sql/workbench/browser/modelComponents/propertiesContainer.component.ts:16]: Oops, good catch. Was left over from my initial attempt which was container-based

File: src/sql/azdata.proposed.d.ts [hunk 3]
```diff
@@ -305,6 +306,41 @@ declare module 'azdata' {
 		required?: boolean;
 	}
 
+	/**
+	 * A property to be displayed in the PropertiesContainerComponent
+	 */
+	export interface DisplayProperty {
```
Comment (alanrenmsft) [inline src/sql/azdata.proposed.d.ts:312]: this is only related to the PropertiesContainer, maybe naming it PropertiesContainerItem?

**Pull request merged**